components/server/src/data_model/metrics.py

```diff
@@ -390,5 +390,15 @@
             sources=["manual_number", "ojaudit", "sonarqube"],
             tags=[Tag.MAINTAINABILITY],
         ),
+        sentiment=dict(
+            name="Sentiment",
+            description="How are the team members feeling?",
+            unit=Unit.NONE,
+            direction=Direction.MORE_IS_BETTER,
+            target="10",
+            near_target="8",
+            sources=["manual_number"],
+            tags=[Tag.PROCESS_EFFICIENCY],
```

Comment:
Hmm, seems weird to classify sentiment as "process efficiency" metric. Maybe "process quality"? Or no tags at all.